plugin/path_role_set.go

```diff
@@ -270,6 +215,9 @@ func (b *backend) pathRoleSetCreateUpdate(ctx context.Context, req *logical.Requ
 	}
 	name := nameRaw.(string)
```

Comment:
Since `name` is a path parameter, it's presence will be guaranteed by the framework (i.e. the `if !ok {...}` will never be hit). So you can optionally do: `name := d.Get("name").(string)`